parsers/link-design-tokens/README.md

```diff
@@ -1,15 +1,30 @@
 # Link design tokens
 
 ## Description
+This parser helps you have design tokens referencing other ones.
+It replaces absolute values by their potential corresponding design token.
 
-This parser replace absolute values by the design token corresponding.
+Use this parser to make your borders, shadows and gradient reference other design token types like measurement or color.
 
-If you have a border which include a width of `{ measure: 3, unit: 'px' }` and a measurement with the same value.
-The parser will replace the value of `width` by the entire measurement token.
+For instance, a border design token is composed of:
+- a border width (a measurement design token)
+- a border type
+- a border color (a color design token)
+
+The border width and the border color absolute values will be replaced by their corresponding design tokens.
+
+**ℹ️ Good to know**
+1. This parser is meant to be used before the [`to-theme-ui`](https://github.com/Specifyapp/parsers/tree/master/parsers/to-theme-ui) parser.
+2. In the future, it will also be used by the [`to-css-custom-properties`](https://github.com/Specifyapp/parsers/tree/master/parsers/to-css-custom-properties) to allow an output like this for example: <br>`--border-token: var(--measurement-token) solid var(--color-token);`
```

Comment:
1. Prefere a sentence like: This parser is useful when you use it before ...
2. Do not use a readme as a roadmap. Remove point 2

Reply:
I removed point 2. However, we don't have a blog so we don't have anywhere else to explain how a parser can be used.

Reply:
Create another support for that.